resources/views/filament/footer.blade.php

```diff
@@ -35,13 +35,13 @@
 
     <div class="mt-4 flex w-full justify-center">
         <img
-            class="hidden h-5 dark:block"
-            src="{{ Vite::asset('resources/images/canyon-logo-dark.png') }}"
+            class="hidden h-7 dark:block"
+            src="{{ Vite::asset('resources/svg/CGBS_Logo_FullColor_Dark.svg') }}"
             alt="{{ config('app.name') }}"
         />
         <img
-            class="block h-5 dark:hidden"
-            src="{{ Vite::asset('resources/images/canyon-logo-light.png') }}"
```

Comment:
It looks like this old png may still be being used in one other place that needs to be replaced: `app-modules/service-management/src/Http/Controllers/ServiceRequestFeedbackFormWidgetController.php`

Once you have replaced that please confirm that this and the dark mode png are not being used anywhere else and then remove them.

Reply:
The old images were not removed. Are you keeping them because they are used somewhere you did not change?